nx-dev/ui-blog/src/lib/more-blogs.tsx

```diff
@@ -1,11 +1,58 @@
 import Link from 'next/link';
 import { BlogPostDataEntry } from '@nx/nx-dev/data-access-documents/node-only';
 import { BlogAuthors } from './authors';
+import {
+  ComputerDesktopIcon,
+  BookOpenIcon,
+  MicrophoneIcon,
+  CubeIcon,
+  AcademicCapIcon,
+  ChatBubbleOvalLeftEllipsisIcon,
+  XCircleIcon,
+} from '@heroicons/react/24/outline';
 
 export interface MoreBlogsProps {
   blogs: BlogPostDataEntry[];
 }
 
+const filters = [
```

Comment:
should we deduplicate this one with the one in the `blog-container.tsx`? Otherwise if we add support for a new topic we might miss to update either of the two.